Source/ULibs.UlibsProjectTemplate/Class1.cs

```diff
@@ -1,4 +1,5 @@
-﻿/***using System.Diagnostics.CodeAnalysis;***/
+﻿#nullable enable

```

Comment:
Do we need this file since there is nothing in it (and its called `Class1.cs`)?

Reply:
It's serving as an example of how the code coverage should be excluded and the namespace gets modified according to the project that imports it. Assuming that still works for the new-style source packages...